### DISCUSSION TITLE
Can't redeem my tokens (solved)

### DISCUSSION THREAD

**Discussion by warlog** [wormhole-foundation/wormhole#2431] [Problems / bugs] [answered]
Hi, I'm trying to redeem a token but getting Error: requested VAA not found in store

TX: https://etherscan.io/tx/0x3d621f068286e843fa66506428f18a28358430308de069dee2312145fa20d80c

Help pls

**Comment by evan-gray** [comment, marked as answer]
Seems like you were successful in the redemption as this transaction has been redeemed 🙂
